precompiles/assets/assets.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity >=0.8.17;
+
+/// @dev The Assets contract's address.
+address constant ASSETS_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000804;
+
+/// @dev The Assets contract's instance.
+IAssets constant ASSETS_CONTRACT = IAssets(
+    ASSETS_PRECOMPILE_ADDRESS
+);
+
+/// @author Exocore Team
+/// @title Assets Precompile Contract
+/// @dev The interface through which solidity contracts will interact with assets module
+/// @custom:address 0x0000000000000000000000000000000000000804
+interface IAssets {
```

Comment:
we'd better name this sol file with its contract name `IAssets.sol`, which more conforms to convention

Reply:
Thanks, will rename it.